src/test/java/com/ft/jerseyhttpwrapper/ResilientClientTest.java

```diff
@@ -126,7 +126,7 @@ public void shouldHonourDefaultTimeout() {
         aResponse()
             .withStatus(200)
             .withHeader(CONTENT_TYPE, "text/plain")
-            .withFixedDelay(501)
+            .withFixedDelay(510)
```

Comment:
What's the reason behind changing this value?

Reply:
this one is ok during circleci build - there is no drama with it